src/Package/Impl/DataInspect/VariableView.xaml.cs

```diff
@@ -4,20 +4,31 @@
 using System;
 using System.Collections.Generic;
 using System.ComponentModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
 using System.Windows.Controls;
+using Microsoft.Common.Core;
 using Microsoft.R.Debugger;
+using Microsoft.R.Host.Client;
 using Microsoft.R.Support.Settings.Definitions;
 using Microsoft.VisualStudio.R.Package.DataInspect.Definitions;
+using Microsoft.VisualStudio.R.Package.Repl;
 using Microsoft.VisualStudio.R.Package.Shell;
+using static System.FormattableString;
 
 namespace Microsoft.VisualStudio.R.Package.DataInspect {
     public partial class VariableView : UserControl, IDisposable {
         private readonly IRToolsSettings _settings;
         ObservableTreeNode _rootNode;
-        VariableSubscription _globalEnvSubscription;
+        IREnvironmentProvider _environmentProvider;
+        IRSession _rSession;
+        DebugSession _debugSession;
 
         const string GlobalEnvironmentName = "Global Environment";
 
+        private static REnvironmentCollection _defaultEnvironments = new REnvironmentCollection() { new REnvironment(Package.Resources.VariableExplorer_EnvironmentName) };
```

Comment:
Where is `VariableExplorer_EnvironmentName` defined? And why does it even need a default collection?